wiki/Options/en.md

```diff
@@ -529,609 +166,141 @@ Note: Your current playing song on your jukebox will be used.
-</tr>
-<tr class="odd">
-<td><p>Use skin's sound samples</p></td>
-<td><p>Use the custom sound effects of the currently selected skin. Otherwise, osu! by peppy sound sample will be used.</p></td>
-</tr>
-<tr class="even">
-<td><p>Use Taiko skin for Taiko mode</p></td>
-<td><p>Uses <a href="http://osu.ppy.sh/forum/viewtopic.php?f=15&amp;t=662">LuigiHann's Taiko skin</a> when playing Taiko mode. Disable it when your current skin have Taiko-specific elements.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Always use skin cursor</p></td>
-<td><p>Your selected skin cursor (the cursor your mouse controls) will always be used regardless of the beatmap-specific cursor modification.</p></td>
-</tr>
-<tr class="even">
-<td><p>Cursor Size</p></td>
-<td><p>Adjust your cursor size. Default is 1x and range around 0.5x - 2.0x.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Automatic Cursor Sizing</p></td>
-<td><p>Adjust Cursor size based on <a href="CS" class="uri" title="wikilink">CS</a> settings of the selected beatmap's difficulty.</p></td>
-</tr>
-<tr class="even">
-<td><p>Use combo colour as tint for slider ball</p></td>
-<td><p>The transparent slider ball will follow the slider's combo colour. Only works if skin is supported.</p></td>
-</tr>
-</tbody>
-</table>
-
-![](OP Input thumb.jpg "fig:OP Input thumb.jpg") Input section
---------------------------------------------------------------
-
-<img src="optionsinput1.jpeg" title="Input Tab" alt="Input Tab" width="300" />
+## Skin section
+
+### Skin
+
+| Name                                     | Effect                                                                                         |
+|------------------------------------------|------------------------------------------------------------------------------------------------|
+| Skin sample image                        | Shows various gameplay elements from the skin. Clicking will cycle through different elements. |
+| Current Skin                             | Change the current skin (default is at the top of the list).                                   |
+| Preview gameplay                         | Start a random song from your collection on autoplay so you can see the skin in action.        |
+| Open current skin folder                 | Open the directory the current skin is stored in.                                              |
+| Export as .osk                           | Export your current skin as a .osk file.                                                       |
+| Ignore all beatmap skins                 | Favor the current skin over pack-in skins supplied by beatmaps.                                |
+| Use skin's sound samples                 | Use the custom sound effects of the currently selected skin.                                   |
+| Use Taiko skin for Taiko mode            | Always use the default skin for Taiko mode.                                                    |
```

Comment:
this doesn't seem right, if this is enabled, then this should force osu! to go through the `Taiko` folder inside the user's selected skin folder (if there is one) to be used as the taiko skin